perf.go

```diff
@@ -337,6 +379,18 @@ func (pr *PerfReader) poll(epollFd int, rings map[int]*perfEventRing, samples ch
 				return
 			}
 
+			if fd == pr.flushCloseFd {
+				for _, ring := range rings {
```

Comment:
`break` here, and put the flush loop after the poll loop?

Reply:
I'm not sure that makes things any clearer?